broker-daemon/broker-rpc/wallet-service/withdraw-funds.js

```diff
@@ -22,8 +22,12 @@ async function withdrawFunds ({ params, relayer, logger, engines }, { WithdrawFu
     throw new PublicError(`No engine available for ${symbol}`)
   }
 
-  const multiplier = currencies.find(({ symbol: configSymbol }) => configSymbol === symbol).quantumsPerCommon
-  const amountInSat = (Big(amount) * multiplier)
+  const multiplier = (currencies.find(({ symbol: configSymbol }) => configSymbol === symbol) || {}).quantumsPerCommon
+  console.log('multiplier', multiplier)
+  if (!multiplier) {
+    throw new PublicError(`No multiplier available for ${symbol}`)
```

Comment:
The `multiplier` language might be confusing. We might be better saying something about how the broker configuration is bad (missing quantumsPerCommon)